gems/nokogiri/CVE-2021-3518.yml

```diff
@@ -0,0 +1,26 @@
+---
+gem: nokogiri
+cve: 2021-3518
+ghsa: v4f8-2847-rwm7
+url: https://bugzilla.redhat.com/show_bug.cgi?id=1954242
```

Comment:
Maybe the primary URL should be `https://github.com/advisories/GHSA-v4f8-2847-rwm7` which discusses nokogiri specifically.